viewer/db.js

```diff
@@ -1730,61 +1750,118 @@ Db.getIndices = async (startTime, stopTime, bounding, rotateIndex) => {
       }
       if (index.startsWith('sessions2-')) { // sessions2 might not have prefix
         index = index.substring(10);
+      } else if (internals.queryExtraIndicesRegex.some(re => re.test(index))) {
+        // extra user-specified indexes from the queryExtraIndices don't have the prefix
+        isQueryExtraIndex = true;
```

Comment:
Check the array of regular expressions matching the `queryExtraIndices` patterns against the given index name. If any of them match, set `isQueryExtraIndex` so we can behave accordingly.